src/pages/FishListPage.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import FishCard from '../components/FishCard.js';
+import Header from '../components/Header.js';
+
+export default function FishList({ fishes }) {
+  return (
+    <PageContainer>
+      <Header />
+      <FishlistContainer>
+        {fishes && fishes.map(fish => <FishCard key={fish.Id} fish={fish} />)}
+      </FishlistContainer>
+    </PageContainer>
+  );
+}
```

Comment:
I would suggest to put the header component in the app and the grid as well. Your FishListPage could then be rendered in the main-section in the App. I guess, it is not wrong what you did here but it seems strange to me that the header is in your main element, especially because it is your only header and basically the main headline for your app.